frontend/apps/marketing/src/app/[brand]/[locale]/layout.tsx

```diff
@@ -7,53 +7,53 @@ import {getGoogleAnalyticsMeasurementId} from '@/config/ga4';
 import OrganizationJsonLd from '@/config/jsonLd/OrganizationJsonLd';
 import {getStage} from '@/config/stage';
 import EnvironmentLoader from '@/providers/environment';
+import LocalizeLoader from '@/providers/localize/LocalizeLoader';
 import NewRelicLoader from '@/providers/newrelic/NewRelicLoader';
 import OneTrustLoader from '@/providers/onetrust/OneTrustLoader';
 import OneTrustProvider from '@/providers/onetrust/OneTrustProvider';
 import {generateBootstrapValues} from '@/providers/statsig/statsig-backend';
 import StatsigProvider from '@/providers/statsig/StatsigProvider';
 
-/**
- * Nested asynchronous layout to temporarily workaround Font Awesome imports going out of order due to CSS Chunking
- *
- * Long term fix: https://codedotorg.atlassian.net/browse/CMS-413
- */
 export default async function Layout({
   children,
   params,
 }: Readonly<{
   children: React.ReactNode;
-  params: Promise<{brand: Brand}>;
+  params: Promise<{brand: Brand; locale: string}>;
 }>) {
   const syncParams = await params;
-  const {brand} = syncParams;
+  const {brand, locale} = syncParams;
 
   const googleAnalyticsMeasurementId = getGoogleAnalyticsMeasurementId(brand);
   const statsigBootstrapValues = await generateBootstrapValues();
   const statsigClientKey = process.env.STATSIG_CLIENT_KEY;
 
   return (
-    <>
-      <EnvironmentLoader />
-      <NewRelicLoader />
-      <OneTrustLoader brand={brand} />
+    <html lang={locale}>
```

Comment:
This should add direction support for RTL languages in Contentful (related to this ticket: [CMS-817](https://codedotorg.atlassian.net/browse/CMS-817)): 
```suggestion
    <html lang={locale} dir={['ar', 'fa'].includes(locale) ? 'rtl' : 'ltr'}>
```
----

https://github.com/user-attachments/assets/f268a13f-c534-4a93-983f-78d93db98343

Reply:
I did a config approach in  [aefdead](https://github.com/code-dot-org/code-dot-org/pull/66358/commits/aefdead61ec2be124b41cdb7ada2794b76b9e655)  which should make it easier to apply generally

Reply:
Nice!